regolith/helpers/l_projectahelper.py

```diff
@@ -152,6 +161,11 @@ def sout(self):
                                                                                           p.get("description"),
                                                                                           p.get("lead"), members,
                                                                                           collaborators))
+        if rc.grp_by_lead:
+            for key, values in grp_by_lead.items():
+                print(key + ':')
```

Comment:
at this point you should be getting in the habit of using f-strings, not concatenating strings like this.